doc/Changelog.md

```diff
@@ -3,6 +3,8 @@ Changelog {#Changelog}
 
 # git master {master)
 
+# Release 0.3.0 (09-11-2015) {#release_003000}
+
 * [6](https://github.com/BlueBrain/Monsteer/pull/6):
```

Comment:
Remove the anchors on the release headings (see other projects) - They look ugly on github and don't add much for doxygen.